frontend/ui-pablo/app/components/Organisms/Wallet/PolkadotConnect.tsx

```diff
@@ -49,7 +49,7 @@ const Status = () => {
               value: asset.assetId,
               label:
               balance.lte(1000)
-                  ? balance.toString()
+                  ? balance.toFixed(2).toString()
```

Comment:
@martins87 you don't need to use toString for toFixed. toFixed returns string.

Reply:
Fixed